Makefile

```diff
@@ -287,6 +299,8 @@ docker run -dt --privileged \
 	--add-host "$(REGISTRY_HOST):$(shell $(IP_CMD) $(MASTER_CTR)1 2>/dev/null || echo 127.0.0.1)" \
 	$(DOCKER_IMAGE);
 sleep 2;
+docker exec $(1)$(2) mkdir -p /var/lib/dcos
```

Comment:
> Why are we doing multiple docker execs instead of a single one with a multi-line bash script?

No reason - though I don't see why it is better - performance wasn't really my concern if that is yours. I am happy to change this though.

> Why are we doing docker execs at all instead of just making the docker run a bash script?

I continued the path of using execs here but I can change that if you like. Again, I have no strong preference.

Please let me know how to proceed and I will.